pyproject.toml

```diff
@@ -1,3 +1,20 @@
+[tool.pytest.ini_options]
+addopts = ["-p", "no:asyncio"]
+filterwarnings = [
+    "error",
+    "ignore:The loop argument is deprecated since Python 3.8:DeprecationWarning",
+    'ignore:"@coroutine" decorator is deprecated since Python 3.8:DeprecationWarning',
+    "default:Tried to add marker .* but that test doesn't exist.:RuntimeWarning",
+    "ignore:the imp module is deprecated in favour of importlib.*:DeprecationWarning",
+    "ignore:'AbstractChildWatcher' is deprecated.*:DeprecationWarning"
+]
+junit_family = "xunit2"
+xfail_strict = true
+
+[tool.flake8]
+max-line-length = 99
+extend-ignore = ['D', 'E402', 'E731', 'E127', 'E502', 'E123', 'W503']
```

Comment:
Is this configuration actually used by a tool run by CI?

Reply:
It was previously in `setup.cfg` so I figured I should move it over. CI doesn't currently do any linting or formatting except yapf. I'd rather not try to change that in this PR, it's already big enough.